ChangeLog.d/issue1792.txt

```diff
@@ -0,0 +1,4 @@
+Bugfix
+   * Fix a bug in EC J-PAKE that would cause it fail when the payload is all-
+     bits-zero.
+     Found by Gilles Peskine, reported in #1792.
```

Comment:
We not longer cite who reported a bug in the ChangeLog, and we never did for team members. Please only mention the issue number.